Makefile

```diff
@@ -131,8 +131,10 @@ docker:
 	@git archive --format=tar.gz HEAD | docker build -t hypothesis/hypothesis:$(DOCKER_TAG) -
 
 .PHONY: docker-ws
-docker-ws:
+docker-ws: docker
+	docker tag hypothesis/hypothesis:dev hypothesis/hypothesis:latest 
 	docker build -t hypothesis/hypothesis-ws:$(DOCKER_TAG) h/streamer 
+	docker rmi hypothesis/hypothesis:latest
```

Comment:
Without this retag dance, we go and pull the latest image from Docker hub, which is mega confusing as it's got nothing to do with the local code.

The other way to get around this would be to set `DOCKER_TAG` to `latest` instead of `dev` which forces this behavior.